main.js

```diff
@@ -1,39 +1,113 @@
+/* eslint-disable no-unused-vars */
```

Comment:
You can delete this comment now that you are using the `cars` variable in your code.